Packs/RandomImages_VideosAndAudio/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Random Images, Videos and Audio",
+    "description": "This packs automation scripts and / or integrations that can pull random photos, videos or audio.\n\nThere is nothing special about this pack. It was designed to lighten up dashboards and incident layouts if so desired.\n\nMore and more scripts and automations will be added over time.\n\nSimply add the automation script to a dynamic-section in an incident layout, or into a script based widget in a dashboard.",
```

Comment:
Please consider adding that to the pack README

Reply:
@Itay4  Just FYI: When you submit a new pack through the marketplace, there is no way to enter anything into the README.